lib/pollbooth.rb

```diff
@@ -1,3 +1,4 @@
+require 'active_support'
```

Comment:
Apparently you have to require this first now (they made it so that it only loads the bare minimum).

http://guides.rubyonrails.org/active_support_core_extensions.html